### PR TITLE
sheets page updated

### PR DESCRIPTION
**Pull request by venkatesh-leapsquarelabs**

### CODE REVIEW

File: src/features/sheetList/ui/sheetList.js [hunk 1]
```diff
@@ -214,36 +205,22 @@ export default function SheetList() {
...
-                  name: item.process.process_name,
-                };
-              },
-            }}
-          /> */}
           <SingleSelectDropdown
             value={{
               buttonStyle: {
-                borderRadius: "11px",
+                borderRadius: "0px 10px 10px 0px",
```
Comment (Tejveer182) [inline src/features/sheetList/ui/sheetList.js:213]: cannpt be targetted from css?

File: src/features/sheetList/ui/sheetList.js [hunk 2]
```diff
@@ -177,34 +177,25 @@ export default function SheetList() {
...
         <div>
           <CustomToastContainer />
         </div>
-        <div style={{ flexGrow: 1, marginLeft: "20px" }}>
+        <div style={{ flexGrow: 1 }}>
           <SingleSelectDropdown
             value={{
               buttonStyle: {
-                borderRadius: "11px",
+                borderRadius: "10px 0px 0px 10px",
```
Comment (Tejveer182) [inline src/features/sheetList/ui/sheetList.js:187]: cannot from css class?

File: src/features/sheets/components/newSheetDetailsModal/ui/newSheetDetailsModal.js [hunk 3]
```diff
@@ -207,81 +208,133 @@ export default function NewSheetDetailsModal(props) {
...
-                )}
+                <SingleSelectDropdown
+                  value={{
+                    state: products,
+                    buttonText: selectedProduct.id
+                      ? selectedProduct.name
+                      : "Link Supplier",
+
+                    buttonStyle: {
+                      borderRadius: "10px",
```
Comment (Tejveer182) [inline src/features/sheets/components/newSheetDetailsModal/ui/newSheetDetailsModal.js:219]: css class?

File: src/features/sheetList/ui/sheetList.js (hunk 2 above)
Comment (venkatesh-leapsquarelabs) [reply src/features/sheetList/ui/sheetList.js:187]: SingleSelectDropDown has used so many places so, one option i have is send the styles as prop

File: src/features/sheetList/ui/sheetList.js (hunk 1 above)
Comment (venkatesh-leapsquarelabs) [reply src/features/sheetList/ui/sheetList.js:231]: noo broooo

File: src/features/sheetList/ui/sheetList.js (hunk 2 above)
Comment (venkatesh-leapsquarelabs) [reply src/features/sheetList/ui/sheetList.js:206]: noo

File: src/features/sheets/components/newSheetDetailsModal/ui/newSheetDetailsModal.js (hunk 3 above)
Comment (venkatesh-leapsquarelabs) [reply src/features/sheets/components/newSheetDetailsModal/ui/newSheetDetailsModal.js:219]: noo

**Comment by Tejveer182** [comment]
Verify with Figma once and make changes and push them.

File: src/features/sheetList/ui/sheetList.js [hunk 4]
```diff
@@ -59,23 +60,15 @@ export default function SheetList() {
   useEffect(() => {
     (async () => {
```
Comment (Tejveer182) [inline src/features/sheetList/ui/sheetList.js:61]: can we remove async as it won't work in an useeffect

**Pull request merged**